src/city.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, name, colour):
         self.has_lab = False
         self.colour = colour
         self.infection_levels = {}
-        self.distance = 999
+        self.distance = 999 # TODO unused?
```

Comment:
Yes, unused.
To be removed in the next cleanup branch.
Nothing to do here right now.